### PR TITLE
Enhancement/tooling fixes and upgrades

### PR DESCRIPTION
**Pull request by ThisIsBrady**
This PR resolves issues with the project's toolchain and attempts to standardize code format and style. 

Style is enforced through the GitHub Actions test, and formatting is enforced by Prettier, which is recommended to run on save in VS Code. The recommended plugins are also included in the workspace and will prompt the user to install on first open.

This PR upgrades the project to Boilerplate 2.3.0, the last update for version 2.

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -6,8 +6,8 @@ daysUntilClose: 7
 exemptLabels:
   - pr
   - discussion
-  - e2e
   - enhancement
+  - bug
```
Comment (ThisIsBrady) [inline .github/stale.yml:10]: Add 'feature' exemption

File: src/Core/components/ControlScheme.tsx [hunk 2]
```diff
@@ -221,27 +225,27 @@ class ControlScheme extends Component<IProps, IState> {
       // this is a callback for when the setState finished updating it clears the set interval,
       // so that after setState is finished it is able to create a new interval and assign it
       () => {
-        if (this.state.functionality[config].toggled === "On") {
-          this.setState({
+        if (this.state.functionality[config].toggled === 'On') {
```
Comment (ThisIsBrady) [inline src/Core/components/ControlScheme.tsx:228]: Fix Nested SetState

File: src/Core/components/ControlScheme.tsx [hunk 3]
```diff
@@ -254,126 +258,129 @@ class ControlScheme extends Component<IProps, IState> {
         },
       },
       () => {
-        if (this.state.functionality[config].toggled === "On") {
-          this.setState({
+        if (this.state.functionality[config].toggled === 'On') {
```
Comment (ThisIsBrady) [inline src/Core/components/ControlScheme.tsx:261]: Fix nested SetState

File: src/RAM/Arm/components/Angular.tsx [hunk 4]
```diff
@@ -176,16 +180,16 @@ class Angular extends Component<IProps, IState> {
...
-      selectedPosition = this.state.positionName
+      selectedPosition = this.state.positionName;
     }
 
     this.setState(
       {
         selectedPosition,
         addingPosition: false,
-        positionName: "", // reset name in modal text box for next use
+        positionName: '', // reset name in modal text box for next use
```
Comment (ThisIsBrady) [inline src/RAM/Arm/components/Angular.tsx:192]: Fix nested setstate

File: src/RAM/Arm/components/ControlMultipliers.tsx [hunk 5]
```diff
@@ -95,17 +99,17 @@ class ControlMultipliers extends Component<IProps, IState> {
         },
       },
       () => {
-        controlMultipliers = this.state.controlMultipliers
+        controlMultipliers = this.state.controlMultipliers;
```
Comment (ThisIsBrady) [inline src/RAM/Arm/components/ControlMultipliers.tsx:102]: Fix nested setState

**Review by UhOhDonovan — APPROVED** [review]
All of these changes allow BaseStation to work properly, and are consistent with our general purpose this year of organization and readability.

**Pull request merged**